recommender/mock-recommendation-app.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/example-shopping
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import * as express from 'express';
+import {Server} from 'http';
+const recommendations = require('./recommendations.json');
+
+export function createRecommendationServer(port: number = 3001) {
```

Comment:
It should just take `(options: http.ServerOptions = {port: 3001})`

Reply:
No other options are configurable in the app and I can't find a `http.ServerOptions` type.

Reply:
Oh, we only have `https.ServerOptions`.

Reply:
Keeping it an `options` object will allow us to configure it to be https too.

Reply:
So I should set the type to be `https.ServerOptions`?